charts/nfs-subdir-external-provisioner/values.yaml

```diff
@@ -2,7 +2,7 @@ replicaCount: 1
 strategyType: Recreate
 
 image:
-  repository: k8s.gcr.io/sig-storage/nfs-subdir-external-provisioner
+  repository: registry.k8s.io/sig-storage/nfs-subdir-external-provisioner
```

Comment:
please update the default value in the chart readme

https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/blame/master/charts/nfs-subdir-external-provisioner/README.md#L55